FreeSurfer/LongitudinalFreeSurferPipeline.sh

```diff
@@ -187,23 +187,22 @@ ${HCPPIPEDIR}/show_version
 
 #processing code goes here
 echo "parallel mode: $parallel_mode"
+fslsub_queue=NONE
+max_jobs=2
+
 case $parallel_mode in
 	FSLSUB)
-    if [ -n "$queue" ]; then 
-		  queuing_command="$FSLDIR/bin/fsl_sub -q $queue"
-    else
-      queuing_command="$FSLDIR/bin/fsl_sub"
-    fi
-		;;    
+    if [ -n "$queue" ]; then fslsub_queue=$queue; fi	
+    ;;    
 	BUILTIN)
-		queuing_command="par_addjob"
-		max_jobs=$parallel_mode_param
-		;;
+		max_jobs=$max_jobs
```

Comment:
One variable name stores only one thing, this isn't going to allow you to set it to something else.  `parellel.shlib` already has code to detect the number of physical processors, why are you setting a default?

Reply:
Are you talking about max_jobs? I'd prefer to have manual control over this as this might not be directly related to number of available processors.

Reply:
`a=$a` does nothing.  This line shouldn't exist.

As a separate issue, `--max-jobs` has its default set to `4`, rather than empty string (autodetect - also supported for "-1").  You don't know how many cores the user will have or want to use, so a hardcoded default of 4 is strange.  Having manual control when desired doesn't need to set a default that avoids the automatic logic.

You can/should also let parallel.shlib use its existing handling to yell at the user for an invalid setting, so the next line testing `(( max_jobs < 1))` should also be deleted, as it would prevent either autodetect value from working.

Reply:
Commented on this above. Done.